dist_config.py

```diff
@@ -10,8 +10,7 @@
 # See descriptions of WHEEL_LINUX_CONFIGS for details.
 SDIST_CONFIG = {
     'image': 'nvidia/cuda:10.2-devel-centos6',
-    # This image contains cuDNN and NCCL.
-    'verify_image': 'nvidia/cuda:11.4.2-cudnn8-devel-{system}',
+    'verify_image': 'nvidia/cuda:11.5.0-devel-{system}',
```

Comment:
I can see no CUDA 11.5 image contains cuDNN and NCCL for now in https://gitlab.com/nvidia/container-images/cuda